ToNote/Controls/ExtendedRichTextBox.cs

```diff
@@ -39,5 +98,18 @@ public void ReadFromFile(string file)
 
             CurrentFile = file;
         }
+
+        public void TrackKeyword(string keyword, Action action)
+        {
+            var keywordaction = new KeywordAction() { Keyword = keyword, Action = action };
+            if (!_trackedKeywords.Contains(keywordaction))
```

Comment:
I think this check is redundant. `KeywordAction` contains an `Action`. For a comparison between two `KeywordAction` instances to return true, they would have to reference the same action (default reference equality comparison). One would have to go out of their way to trigger this check, for example even just copy-paste duplicating the "note" keyword registering code does not trigger this condition.
If you think that one shoud only be able to register one action per keyword, overload the equality comparison operators for `KeywordAction`. 
However, I don't really see why you shouldn't be able to register multiple actions per keyword (unless maybe there could be some issues with execution order?). If we allow this, you can just remove this check, but then also don't forget to remove `return` from the foreach where you search for maching keywords.